android/src/main/java/com/reactnativestripesdk/AuBECSDebitFormViewManager.kt

```diff
@@ -1,35 +1,44 @@
 package com.reactnativestripesdk
 
-import com.facebook.react.bridge.ReadableMap
+import com.facebook.react.bridge.Dynamic
 import com.facebook.react.module.annotations.ReactModule
 import com.facebook.react.uimanager.SimpleViewManager
 import com.facebook.react.uimanager.ThemedReactContext
 import com.facebook.react.uimanager.annotations.ReactProp
+import com.facebook.react.viewmanagers.AuBECSDebitFormManagerDelegate
+import com.facebook.react.viewmanagers.AuBECSDebitFormManagerInterface
+import com.reactnativestripesdk.utils.asMapOrNull
 
 @ReactModule(name = AuBECSDebitFormViewManager.REACT_CLASS)
-class AuBECSDebitFormViewManager : SimpleViewManager<AuBECSDebitFormView>() {
+class AuBECSDebitFormViewManager :
+  SimpleViewManager<AuBECSDebitFormView>(),
+  AuBECSDebitFormManagerInterface<AuBECSDebitFormView> {
+  private val delegate = AuBECSDebitFormManagerDelegate(this)
```

Comment:
For android this is a simple example of the changes needed:

- Implement the codegen interface
- Add the delegate
- UnsafeMixed generates Dynamic type instead of ReadableMap, so I made a extension method to easily convert between the two.